src/udiskslinuxswapspace.c

```diff
@@ -244,22 +260,41 @@ swapspace_stop_on_job_completed (UDisksJob   *job,
 }
 
 static gboolean
+stop_job_func (UDisksThreadedJob  *job,
+               GCancellable       *cancellable,
+               gpointer            user_data,
+               GError            **error)
+{
+
+  UDisksObject *object = NULL;
+  UDisksBlock *block = NULL;
+  gchar *device = NULL;
+  gboolean ret = FALSE;
+
+  object = UDISKS_OBJECT (g_dbus_interface_get_object (G_DBUS_INTERFACE (user_data)));
+  block = udisks_object_peek_block (object);
```

Comment:
From the docs:
```
It is not safe to use the returned object if you are on another thread than the one where the GDBusObjectManagerClient or GDBusObjectManagerServer for object is running.
```
``udisks_object_get_block ()`` should be used instead as that increases the ref counter.